### DISCUSSION TITLE
Minio error full database  - storage 30GB free space but I can't upload anything

### DISCUSSION THREAD

**Discussion by Mendesz08** [minio/minio#18722] [Q&A]
good morning !

I'm having an error on my minio server, it has 100 GB of space but I'm only using 70% of it and when I try to upload any file it gives me this error:  Error: Storage backend has reached its minimum free drive threshold. Please delete a few objects to proceed.

Is there any configuration to increase the limit to 80 or 90%?

And if so, what would be the best choice to make?

att,

![image](https://github.com/minio/minio/assets/144833757/da3cba9d-e54e-43fa-8306-ff5557672c34)

**Comment by klauspost** [comment]
> And if so, what would be the best choice to make?

Don't use this small volumes.

**Reply by Mendesz08** [reply]
I don't understand, could you explain it better?

the application is in docker.

Can you tell me if there is any configuration for docker?

**Reply by Mendesz08** [reply]
![image](https://github.com/minio/minio/assets/144833757/f1a94318-6ede-4ac6-b2c2-122e2b40a76a)

**Reply by Mendesz08** [reply]
Do you think this could be it?

**Reply by Mendesz08** [reply]
![image](https://github.com/minio/minio/assets/144833757/416eabf8-3579-4d46-9348-2c47ae7a8a70)

**Reply by harshavardhana** [reply]
Correct you need free inodes to operate

**Reply by Mendesz08** [reply]
How do I increase my inodes?
